app/graphql/types/customers/usage/charge_filter.rb

```diff
@@ -6,7 +6,7 @@ module Usage
       class ChargeFilter < Types::BaseObject
         graphql_name 'ChargeFilterUsage'
 
-        field :id, ID, null: false, method: :charge_filter_id
+        field :id, ID, null: true, method: :charge_filter_id
```

Comment:
why?

Reply:
For the default values (the values attached to the charge itself), we are generating a fee without a persisted charge filter. Since the front-end need to know that this fee is the one matching the default values, and some other attributes (units, unit_amount...), we are returning a "virtual" charge filter with ID = nil